addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -51,6 +95,8 @@ def init(self):
                     JOIN res_partner P ON (S.partner_id = p.id)
                     JOIN res_users U ON (U.partner_id = P.id)
                     JOIN im_livechat_channel_im_user O ON (O.user_id = U.id and O.channel_id = L.id)
-                GROUP BY C.id, C.name, C.livechat_channel_id, L.name, C.create_date, C.uuid, S.partner_id
+                    LEFT JOIN res_country Country ON (Country.id = C.country_id) 
```

Comment:
Don't need to include this table in the FROM, as we don't use any value from res_country

Reply:
Yes, we use the country name